JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/core/EScikitLearnProblemType.java

```diff
@@ -2,29 +2,25 @@
 
 public enum EScikitLearnProblemType {
 
-	REGRESSION("--regression", "sklearn/scikit_template_classification.twig.py"), CLASSIFICATION("", "sklearn/scikit_template_classification.twig.py", "tpot"), RUL("--rul", "sklearn/scikit_template_rul.twig.py"), FEATURE_ENGINEERING("--fe",
-			"sklearn/scikit_template_rul.twig.py");
+	REGRESSION("regression"), //

```

Comment:
required modules missing

Reply:
We have some common required modules defined in the AScikitLeranWrapper ("arff", "numpy", "json", "pickle", "os", "sys", "warnings", "scipy", "sklearn", "pandas", "xgboost"), so here only additional ones have to be defined. Are you missing specific ones?